### DISCUSSION TITLE
数据集内有不同尺寸的图片，yolo会自动预处理吗？

### DISCUSSION THREAD

**Discussion by Gemini-xuanwu** [ultralytics/ultralytics#17016] [Q&A]
您好，如果我的数据集中图片有着不同的大小格式，如416✖416、1280✖720、254✖254与1920✖1080四种不同大小的格式，yolo会自动识别并进行预处理图片以及相应的标签吗？还是说一定要有一致的格式才行？

**Comment by glenn-jocher** [comment]
@Gemini-xuanwu 您好，YOLO会自动调整不同尺寸的图片以适应模型的输入大小，无需手动统一格式。
